winterfell/webapp/src/styles/signin.styl

```diff
@@ -1,97 +1,216 @@
 @import "standards.styl"
 
+backgroundUrl = '../icons/veteran-dog-bg.jpg'
```

Comment:
ah yea. I was using a super big one. I was able to create a smaller scaled one that is 193KB. I may start purchasing a few photos that are licensed which we can legally use. I found a pretty good site that has really good veteran-related photos. Might start using those.